client/tests/integration/smartcontracts/validator_with_custom_token/src/lib.rs

```diff
@@ -2,6 +2,11 @@
 //! [`token::CanControlDomainLives`] permission token.
 //!
 //! It also doesn't have [`iroha_validator::default::domain::tokens::CanUnregisterDomain`].
+//!
+//! In migration it replaces [`iroha_validator::default::domain::tokens::CanUnregisterDomain`]
+//! with [`token::CanControlDomainLives`] for all accounts.
+//! So it doesn't matter which domain user was able to unregister before migration, they will
+//! get access to control all domains. Remember that this is just a test example.
```

Comment:
This feels like it needs a more thorough explanation in the tutorial repo. We've accumulated a lot of material on the runtime validator/executor, so we should update the docs. I'd recommend doing it sooner rather than later.

Reply:
That is unfortunately true and some work will need to be duplicated. But we expect that either RC18, RC19 or RC20 are going to be LTS candidates. This functionality is going to be utilised heavily.